sequence/sequences/compose/arith.py

```diff
@@ -0,0 +1,74 @@
+from sequence.core.core import Sequence
+
+import numpy as np
+
+
+class Arith(Sequence):
+    """Sequences combined via arithmetic operation, e.g., addition, multiplication.
+
+    Parameters
+    ----------
+    sequences : List[Sequence]
+        List of sequences to combine.
+    operation : None, str, function, or numpy ufunc, optional, default = None = mean
```

Comment:
what is `default = None = mean`? Actually `operation` as no default value, right?

Reply:
I see, that is an oversight. Also, perhaps "addition" is a more sensible default.

Reply:
Or no default? As the class should not be used directly by the final user or?

Reply:
why not? A custom `operation` can be provided. Of course dunders are more convenient for most use cases.